src/vmm/src/utilities/mock_resources/mod.rs

```diff
@@ -55,10 +56,17 @@ impl MockBootSourceConfig {
     }
 }
 
+impl Default for MockBootSourceConfig {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
 #[derive(Default)]
 pub struct MockVmResources(VmResources);
 
 impl MockVmResources {
+    ///
```

Comment:
Is this intentional?

Reply:
I forgot to remove it, I will do so in the next patch.